bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/contract/patient/PatientSearchParameters.java

```diff
@@ -71,6 +73,14 @@ public void setName(String name) {
         this.name = name;
     }
 
+    public String getGender() {
```

Comment:
If we add this field here we also need to make sure it's respected by existing API methods that use PatientSearchParameters.
(It would be okay to skip the gender field when searching for similar patients, if your timeline to finish this is tight.)

Reply:
@djazayeri what do you mean by respected?